spec/models/user_spec.rb

```diff
@@ -965,10 +965,12 @@
       end
 
       it 'destroys SuspendedEmail records for each email address' do
-        email_address = user.email_addresses.last
+        email_addresses = user.email_addresses
+        email_address = email_addresses.last
+        expect(email_addresses.count).to eq 1
         expect { user.reinstate! }.
-          to(change { SuspendedEmail.find_with_email(email_address.email) }.
-            from(email_address).to(nil))
+          to(change { SuspendedEmail.find_with_email(email_address.email)&.email_address }.to(nil))
```

Comment:
Let's remove the extra `&.`
```suggestion
          to(change { SuspendedEmail.find_with_email(email_address.email) }.to(nil))
```